### ISSUE TITLE
upgrade pronto-reek to reek 3.3.0

### ISSUE THREAD

**Issue by tansaku** [mmozuras/pronto-reek#9]
reek is currently on 3.3.0  - is there any way to have pronto-reek use this @mmozuras ?

**Comment by tansaku** [comment]
I tried upgrading in my fork, but got this error:

```
/Users/tansaku/.rvm/gems/ruby-2.2.2/bundler/gems/pronto-reek-dd40912221be/lib/pronto/reek.rb:14:in `run': uninitialized constant Reek::Core (NameError)
```

**Comment by mmozuras** [comment]
Done! And released as `0.4.3` :smile:
